ui/js/compareperf.js

```diff
@@ -5,12 +5,23 @@
 "use strict";
 
 perf.controller('CompareChooserCtrl', [
-  '$state', '$stateParams', '$scope', 'ThRepositoryModel', 'ThResultSetModel',
-  function CompareChooserCtrl($state, $stateParams, $scope,
-                              ThRepositoryModel, ThResultSetModel) {
+  'phCompareDefaultNewRepo', 'phCompareDefaultOriginalRepo', '$state', '$stateParams',
+   '$scope', 'ThRepositoryModel', 'ThResultSetModel',
+  function CompareChooserCtrl(phCompareDefaultNewRepo, phCompareDefaultOriginalRepo, $state, $stateParams,
+                              $scope, ThRepositoryModel, ThResultSetModel) {
     ThRepositoryModel.get_list().success(function(projects) {
       $scope.projects = projects;
-      $scope.originalProject = $scope.newProject = projects[0];
+      $scope.originalProject = getProject(phCompareDefaultOriginalRepo);
+      $scope.newProject = getProject(phCompareDefaultNewRepo);
+
+      function getProject(repoName) {
```

Comment:
This should be declared before `getProject` is used (just after $scope.projects=projects;, I think)